content/blog/2018-10-02-outcomerate.md

```diff
@@ -0,0 +1,60 @@
+---
+slug: "outcomerate"
+title: Honest Assessments of Survey Quality with AAPOR Outcome Rates
+package_version: 1.0.1
+authors:
+  - name: Rafael Pilliard Hellwig
+    url: https://github.com/rtaph
+date: 2018-10-02
+categories: blog
+topicid: 
+tags:
+- r
+- community
+- software
+- review
+- onboarding
+- package
+- outcomerate
+- reproducible-research
+- survey
+output: html_document
+---
+
+## Background
+
+Surveys are ubiquitous in the social sciences, and the best of them are meticulously planned out. Statisticians often decide on a sample size based on a theoretical design, and then proceed to inflate this number to account for "sample losses". This ensures that the desired sample size is achieved, even in the presence of non-response. Factors that reduce the pool of interviews include participant refusals, inability to contact respondents, deaths, and frame inaccuracies. The more non-response, the more a study becomes open to criticism about its veracity.
```

Comment:
can you clarify what "frame inaccuracies" are for an outside reader?

Reply:
Yes, good point. I will add footnote.